r/R/util.R

```diff
@@ -209,8 +209,7 @@ handle_csv_read_error <- function(e, schema) {
         "row, you should supply the argument `skip = 1` to prevent the",
         "header being read in as data."
       )
-    ))
+    ), call = call)
   }
-
-  abort(msg)
+  abort(conditionMessage(e), call = call)
```

Comment:
(Comments go for both functions)

We already have msg above so we can still use it.

We could also edit the `if` above to do `msg <- c(msg, i = ...)` and just have one `abort()`, if you wanted to polish further. 

```suggestion
  abort(msg, call = call)
```